app/views/shared/_header_navbar.html.erb

```diff
@@ -1,10 +1,11 @@
-<nav class="navbar navbar-toggleable-md navbar-inverse bg-inverse topbar" role="navigation">
+<nav class="navbar navbar-expand-md navbar-dark bg-dark topbar" role="navigation">
```

Comment:
I think we might need to change line 41 from `navbar-toggleable-sm` to `navbar-expand-sm` as well.